handlers/test.go

```diff
@@ -408,29 +408,15 @@ func RunOutgoingTestCases(t *testing.T, channel courier.Channel, handler courier
 			clog := courier.NewChannelLogForSend(msg, handler.RedactValues(channel))
 			ctx, cancel := context.WithTimeout(context.Background(), time.Millisecond*10)
 
-			// work out what kind of sending this handler supports.. this is temporary
-			legacyHandler, _ := handler.(courier.ChannelLegacyHandler)
-
 			var externalIDs []string
 			var status courier.StatusUpdate
-			var serr, err error
-
-			if legacyHandler != nil {
-				status, err = legacyHandler.SendLegacy(ctx, msg, clog)
+			var serr error
+			var resNewURN urns.URN
```

Comment:
since there's no more `if` `else` we don't need vars declared before they're assigned